include/zephyr/drivers/clock_control/stm32_clock_control.h

```diff
@@ -107,6 +107,8 @@
 #define STM32_ADC12_PRESCALER	DT_PROP(DT_NODELABEL(rcc), adc12_prescaler)
 #define STM32_ADC34_PRESCALER	DT_PROP(DT_NODELABEL(rcc), adc34_prescaler)
 
+#define STM32_TIMER_PRESCALER	DT_PROP(DT_NODELABEL(rcc), timpre)
```

Comment:
Considering that there are checks `#if defined(STM32_TIMER_PRESCALER)` shouldn't it be wrapped with:
```c
#if DT_NODE_HAS_PROP(DT_NODELABEL(rcc), timpre)
```